lib/SVF-FE/LLVMUtil.cpp

```diff
@@ -91,17 +91,22 @@ bool LLVMUtil::isConstantObjSym(const Value *val)
 /*!
  * Return reachable bbs from function entry
  */
-void LLVMUtil::getFunReachableBBs (const Function * fun, DominatorTree* dt, std::vector<const BasicBlock*> &reachableBBs)
+void LLVMUtil::getFunReachableBBs (const SVFFunction* svffun, std::vector<const BasicBlock*> &reachableBBs)
 {
+    assert(!SVFUtil::isExtCall(svffun) && "The calling function cannot be an external function.");
+    //initial DominatorTree
+    DominatorTree dt;
+    dt.recalculate(*svffun->getLLVMFun());
```

Comment:
Is `getFunReachableBBs` called more than once per function? If so, I think `dt.recalculate` on every call is not the best idea.